### PR TITLE
docs: update README.md file

### PR DESCRIPTION
**Pull request by gouravkrosx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,103 @@
...
+
+## Usage
+
+- Replace `@SpringBootApplication` with `@SpringBootApplication(scanBasePackages = {"<your base package>", "io.keploy.servlet"}).` in your main class.
+
+
+- Configure Environment Variables
+    - `APP_NAME`           (eg: APP_NAME = myApp)
+    - `APP_PORT`           (eg: APP_PORT = 8080)
+    - `KEPLOY_URL`         (eg: KEPLOY_URL = http://localhost:8081)
```
Comment (Sarthak160) [inline README.md:60]: @gouravkrosx make this keploy url with endpoint `/api`.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,103 @@
...
+
+    compile group: 'io.keploy', name: 'keploy-sdk', version: 'N.N.N'
+
+## Usage
+
+- Replace `@SpringBootApplication` with `@SpringBootApplication(scanBasePackages = {"<your base package>", "io.keploy.servlet"}).` in your main class.
+
+
+- Configure Environment Variables
+    - `APP_NAME`           (eg: APP_NAME = myApp)
```
Comment (slayerjain) [inline README.md:58]: instead of `eg`, maybe let's use `default` something like (default APP_NAME=myApp)

File: README.md [hunk 3]
```diff
@@ -1,16 +1,103 @@
...
+or to *build.gradle*:
+
+    compile group: 'io.keploy', name: 'keploy-sdk', version: 'N.N.N'
+
+## Usage
+
+- Replace `@SpringBootApplication` with `@SpringBootApplication(scanBasePackages = {"<your base package>", "io.keploy.servlet"}).` in your main class.
+
+
+- Configure Environment Variables
```
Comment (slayerjain) [inline README.md:57]: This should be bold too

File: README.md [hunk 4]
```diff
@@ -1,16 +1,103 @@
...
+- **Generate testcases**
+    - To generate/capture TestCases set  and run your application.
+        1. Set `KEPLOY_MODE = record`
+        2. Run your application.
+        3. Make some API calls.
+
+- **Run the testcases**
+    - **Note:** Before running tests stop the sample application.
+        - Using environment variable
+            1. Set `KEPLOY_MODE = test`
```
Comment (slayerjain) [inline README.md:73]: here also mention, that this is set by default.

File: README.md [hunk 5]
```diff
@@ -1,16 +1,103 @@
...
 
 
 ## Community support
+
 We'd love to collaborate with you to make Keploy great. To get started:
-* [Slack](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA) - Discussions with the community and the team.
+
+* [Slack](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA) - Discussions with the
+  community and the team.
 * [GitHub](https://github.com/keploy/keploy/issues) - For bug reports and feature requests.
```
Comment (slayerjain) [inline README.md:96]: please use the ref of the java sdk repo

**Review by slayerjain — CHANGES_REQUESTED** [review]
please review my comments

**Pull request merged**